src/main/java/core/basesyntax/strategy/impl/ReturnOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnOperation implements OperationHandler {
+    public void handleOperation(FruitTransaction fruitTransaction, Storage storage) {
+        if (storage.getInventory().containsKey(fruitTransaction.getFruit())) {
+            storage.getInventory().compute(fruitTransaction.getFruit(),
```

Comment:
It's recommended to use the `computeIfPresent` method when the intention is to modify a value if the key is present. This will make the code cleaner and remove the need for the `if-else` block.